cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/ConsentStatusApiModel.kt

```diff
@@ -50,93 +49,46 @@ data class ConsentStatusResp(
 
 @Serializable
 data class CcpaCS(
-    @SerialName("applies")
     val applies: Boolean?,
-    @SerialName("ccpaApplies")
-    val ccpaApplies: Boolean?,
-    @SerialName("consentedAll")
-    val consentedAll: Boolean?,
-    @SerialName("dateCreated")
-    val dateCreated: String?,
-    @SerialName("gpcEnabled")
-    val gpcEnabled: Boolean?,
-    @SerialName("newUser")
-    val newUser: Boolean?,
-    @SerialName("rejectedAll")
-    val rejectedAll: Boolean?,
-    @SerialName("rejectedCategories")
-    val rejectedCategories: List<String>?,
-    @SerialName("rejectedVendors")
-    val rejectedVendors: List<String>?,
-    @SerialName("signedLspa")
-    val signedLspa: Boolean?,
-    @Serializable(with = CcpaStatusSerializer::class)
-    val status: CcpaStatus?,
-    @SerialName("GPPData")
-    @Serializable(with = JsonMapSerializer::class)
-    val gppData: Map<String, JsonElement>? = null,
-    @SerialName("uuid")
-    var uuid: String?,
-    @SerialName("webConsentPayload")
-    val webConsentPayload: JsonObject? = null,
-) {
-
-    val uspstring: String
-        get() = generateCcpaUspString(
-            applies = applies ?: false,
-            ccpaStatus = status,
-            signedLspa = signedLspa,
-        )
-}
+    @SerialName("consentedAll") val consentedAll: Boolean?,
+    @SerialName("dateCreated") val dateCreated: String?,
+    @SerialName("gpcEnabled") val gpcEnabled: Boolean?,
+    @SerialName("newUser") val newUser: Boolean?,
+    @SerialName("rejectedAll") val rejectedAll: Boolean?,
+    @SerialName("rejectedCategories") val rejectedCategories: List<String>?,
+    @SerialName("rejectedVendors") val rejectedVendors: List<String>?,
+    @SerialName("signedLspa") val signedLspa: Boolean?,
+    @SerialName("uspstring") val uspstring: String? = null,
+    @Serializable(with = CcpaStatusSerializer::class) val status: CcpaStatus?,
+    @SerialName("GPPData") @Serializable(with = JsonMapSerializer::class) val gppData: Map<String, JsonElement>? = null,
+    @SerialName("uuid") var uuid: String?,
+    @SerialName("webConsentPayload") val webConsentPayload: JsonObject? = null,
+)
```

Comment:
from my pov, the thing that you refactored here looked more readable and it followed [KCS](https://kotlinlang.org/docs/coding-conventions.html#annotations), but it might be the matter of taste here, so I'm ok with that

Reply:
I'd go ahead and remove the annotations all together, keeping only the ones that are strictly necessary (those whose attribute name differ from the api response). 

@bohdan-go-wombat thanks for sharing the KCS link, it seems that annotations without arguments are allowed to be in the same line as the example:
```kotlin
@Test fun foo() { /*...*/ }
```